apps/voting/contracts/Voting.sol

```diff
@@ -86,9 +90,17 @@ contract Voting is IForwarder, AragonApp {
     * @param _supportRequiredPct Percentage of yeas in casted votes for a vote to succeed (expressed as a percentage of 10^18; eg. 10^16 = 1%, 10^18 = 100%)
     * @param _minAcceptQuorumPct Percentage of yeas in total possible votes for a vote to succeed (expressed as a percentage of 10^18; eg. 10^16 = 1%, 10^18 = 100%)
     * @param _voteTime Seconds that a vote will be open for token holders to vote (unless enough yeas or nays have been cast to make an early decision)
+    * @param _earlyExecutionAllowed Boolean telling whether votes are allowed to be early executed or not
     * @param _overruleWindow Seconds representing the period where a voter will be able to override a representative's decision on a vote
     */
-    function initialize(MiniMeToken _token, uint64 _supportRequiredPct, uint64 _minAcceptQuorumPct, uint64 _voteTime, uint64 _overruleWindow)
+    function initialize(
```

Comment:
Not specific to this PR, but wondering whether we should keep old initializers for backwards compatibility. The old initializer could just use the current initializer and set the new parameters to the default zero value (which would have the same effect as if the app was initialized before the upgrade)

If we don't do that, as most templates get the app base from the latest version in the aragonPM repo, publishing a new version with a different initializer will break the old templates.

Reply:
I completely agree with your point, I actually choose that pattern in the past but I remember someone recommending not to do it. IMO, I think it's best to leave initializers as they are and add a separate setter to configure whatever is needed to ensure we are backwards compatible